platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/reconnect/emergency/EmergencyReconnectTeacher.java

```diff
@@ -88,30 +88,24 @@ public void execute(final Connection connection) {
         try {
             final long round = connection.getDis().readLong();
             final Hash hash = connection.getDis().readSerializable();
-            try (final AutoCloseableWrapper<SignedState> stateWrapper =
-                    stateFinder.find(emergencyStateCriteria(round, hash))) {
-                final SignedState state = stateWrapper.get();
-                if (state != null) {
+            try (final ReservedSignedState stateWrapper =
```

Comment:
It would be nice to rename this to remove "wrapper"

Reply:
renamed